doc/src/manual/parallel-computing.md

```diff
@@ -1,10 +1,265 @@
 # Parallel Computing
 
-This part of the manual details the following types of parallel programming available in Julia.
+For any newcomer to multi-thread and parallel computing it can be useful to first appreciate 
+the different levels of parallelism offered by Julia. We can divide them in three main categories :
+
+1. Green Threading or Julia Coroutines 
+2. Multi-Threading
+3. Multi-Core or Distributed Processing
+
+We will consider first Julia  [Tasks (aka Coroutines)](@ref man-tasks) and other high-level interfaces and 
```

Comment:
Double space after `Julia`

Reply:
Ouch! Yeah two CI tests failed for the wastefullness of blank spaces, hope also found some blank lines at EOF.